routes/weather.ts

```diff
@@ -339,6 +324,10 @@ export const getWateringData = async function( req: express.Request, res: expres
  * @param useJson Indicates if the response body should use a JSON format instead of a format similar to URL query strings.
  */
 function sendWateringError( res: express.Response, error: CodedError, useJson: boolean = false ) {
+	if ( error.errCode === ErrorCode.UnexpectedError ) {
+		console.error( `An unexpected error occurred:`, error );
+	}
+
 	sendWateringData( res, { errCode: error.errCode, scale: 100 } );
```

Comment:
I'm still a bit hazy on why you include `scale: 100` when an error occurs. I thought the legacy Firmware behavior would be to just retain the previous setting. I know this has been discussed so I may just be a bit behind on this.

Reply:
We're this behavior. We've decided that it's safer to default to 100% to make sure that long-term errors don't affect watering schedules for extended periods of time. Also note that on newer firmware versions we'll ignore the returned scale if an error occurs and the last successful watering call was <24 hours ago. If a watering scale hasn't successfully been calculated in >24 hours, we'll default to a 100% scale